device/config/configurator.go

```diff
@@ -0,0 +1,118 @@
+package config
+
+import (
+	"github.com/xshellinc/tools/dialogs"
+	"github.com/xshellinc/tools/lib/ssh_helper"
+)
+
+// Default Configurator constants that are describe a specific configuration option
+const (
+	Locale = iota
+	Keymap
+	Wifi
+	Interface
+	Dns
```

Comment:
__[golint]__
* [golint] const Dns should be DNS

<!-- Comment by [SideCI](https://sideci.com) -->